plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -419,11 +438,10 @@ function ListMenuItem:update()
             }
 
             -- Create or replace corner_mark if needed
-            local wright_bottom_pad_available = math.ceil( (dimen.h - wright[1]:getSize().h) *2/3 )
-            -- We should normally use 1/2 because of CenterContainer, but there's
-            -- some space inside the text widget that we can use for a larger marker
-            if wright_bottom_pad_available ~= corner_mark_size then
-                corner_mark_size = wright_bottom_pad_available
+            local mark_size = math.floor(dimen.h / 6)
```

Comment:
Felt a bit sad to lose that curated formula for a vulgar 1/6 :) but it seems to look OK on the emulator.
Might depend on how you go with the previous comment.

Reply:
Shouldn't make a difference wrt changes above, because the 2-line text still has to fit into the ListMenuItem, so it won't overlap.
Your formula is nice :) but on some settings the dogear would actually overlap the text.